src/plotlydash.py

```diff
@@ -0,0 +1,104 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.express as px
+import redis
+from dash.dependencies import Input, Output
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+
+file = 'colorCount.csv'
+r = redis.Redis(
+    host='localhost', port=6379, db=0)
+def setup():
+    # getting colors totals into a dict from redis
+    color_byte_dict = r.hgetall('color_totals')
+    color_totals_dict = {}
+    for key in color_byte_dict:
+        value = color_byte_dict[key].decode('utf-8')
+        key = key.decode('utf-8')
+        color_totals_dict[key] = int(value)
+
+    # getting colors RGB values into a dict from redis
+    rediscolor_RGB_dict = r.hgetall('colors')
+    colorsRGBdict = {}
+    for key in rediscolor_RGB_dict:
```

Comment:
If you split off this code into a separate function you could avoid repeating it.